nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java

```diff
@@ -181,20 +182,14 @@ public String uploadedDataset(@NotNull DocumentModel dataset) {
                         trainingDataBlob.getFile().length() / (1024 * 1024));
                 batchUpload.upload("0", trainingDataBlob);
 
-                batchUpload = getClient().batchUploadManager()
-                                         .createBatch()
-                                         .enableChunk()
-                                         .chunkSize(CHUNK_100_MB);
+                batchUpload = getClient().batchUploadManager().createBatch().enableChunk().chunkSize(CHUNK_100_MB);
 
                 String batch2 = batchUpload.getBatchId();
 
                 log.info("Uploading Evaluation Dataset of size {} MB", evalDataBlob.getFile().length() / (1024 * 1024));
                 batchUpload.upload("1", evalDataBlob);
 
-                batchUpload = getClient().batchUploadManager()
-                                         .createBatch()
-                                         .enableChunk()
-                                         .chunkSize(CHUNK_100_MB);
+                batchUpload = getClient().batchUploadManager().createBatch().enableChunk().chunkSize(CHUNK_100_MB);
```

Comment:
why straighten it?

Reply:
I don't know.

Reply:
it's nuxeo formatter once again that i apply :)